actors/builtin/miner/miner_actor.go

```diff
@@ -1171,10 +1171,6 @@ func (a Actor) ExtendSectorExpiration(rt Runtime, params *ExtendSectorExpiration
 				builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to load sectors in deadline %v partition %v", dlIdx, decl.Partition)
 				newSectors := make([]*SectorOnChainInfo, len(oldSectors))
 				for i, sector := range oldSectors {
-					if !CanExtendSealProofType(sector.SealProof) {
```

Comment:
This check and abort needs to stay on actors v3 for network version 10 to allow this code to sync network version 10 state changes.  The right path forward is to guard the new validation-free code path with a check that `rt.NetworkVersion >= network.Version(11)`

Reply:
Or alternatively guard the removed code path with `rt.NetworkVersion < network.Version(11)`

Reply:
added rt.NetworkVersion() into the CanExtendSealProofType to solve this.